src/components/ec-input-field/ec-input-field.vue

```diff
@@ -155,6 +156,10 @@ const props = defineProps({
     type: Boolean,
     default: false,
   },
+  autocomplete: {
+    type: String,
```

Comment:
In the task the `autocomplete` is boolean.

Reply:
I think we should change that in the task, because autocomplete can have lots of string values: https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete. I'd like to support everything the HTML can.

Reply:
Agree.